src/grpc.js

```diff
@@ -340,6 +389,28 @@ class LndGrpc extends EventEmitter {
 
     return { isDone, cancel }
   }
+
+  /**
+   * Subscribe to Wallet State Stream
+   * And return the current state
+   * @return {Promise<Object>}.
+   */
+  async subscribeWalletState() {
```

Comment:
I'd rename this to `getWalletState` and remove this subscription since we aren't doing anything with it but logging. Client can do that themselves if they want to monitor the wallet state.

Reply:
Done.